k2/csrc/array_ops.h

```diff
@@ -378,6 +378,24 @@ Array1<T> Arange(ContextPtr c, T begin, T end, T inc = 1);
 void RowSplitsToRowIds(const Array1<int32_t> &row_splits,
                        Array1<int32_t> *row_ids);
 
+
+/*
+  This function returns *part* of a row_ids array (it's intended for situations
+  where computing the entire row_ids array would use too much memory).
+    @param [in] row_splits  Input row_splits vector, of dimension num_rows+1
+    @param [in] start_row   The first row that we want the row_id value for,
+                            with 0 <= start_row < row_splits.Back()
```

Comment:
`0 <= start_row < num_rows` ?

Reply:
if it's `0 <= start_row < row_splits.Back()`, Is `start_index` a better name?

Reply:
you're right, or start_element.  start_index is fine.